sensor/kubernetes/listener/watchers/complianceoperator/complianceoperatorrules.go

```diff
@@ -0,0 +1,58 @@
+package complianceoperator
+
+import (
+	"time"
+
+	"github.com/stackrox/rox/pkg/complianceoperator"
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/sensor/kubernetes/client"
+	"github.com/stackrox/rox/sensor/kubernetes/listener/watchers/utils"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+// NewComplianceOperatorRulesWatcher creates a Compliance Operator RulesWatcher.
+func NewComplianceOperatorRulesWatcher(cli client.Interface) *RulesWatcher {
+	return &RulesWatcher{
+		cli: cli,
+	}
+}
+
+// RulesWatcher is responsible to watch if the CRD for Compliance Operator Rules becomes available in the cluster.
+type RulesWatcher struct {
+	cli client.Interface
+}
+
+// Watch for the availability of the Compliance Operator Rules CRD and calls stopCallback if it becomes available.
+func (w *RulesWatcher) Watch(stop *concurrency.Signal, stopCallback func(string)) {
+	featureDisabledMsg := "Sensor will not watch for the presence of Compliance CRDs. If the Compliance Operator is deployed in the cluster, Sensor will require a manual restart"
+	if env.ComplianceCRDsWatchTimer.DurationSetting() == 0 {
```

Comment:
note: Early feedback
Is it possible to watch the CRDs instead of polling?
If we poll, in worst case a user needs to wait 10 minutes until the integration becomes available.

Reply:
Good point! I'll modify the final code to use an CRD informer in the watchers instead of polling.